docs/guides/high-availability/deploy-infinispan-kubernetes-crossdc.adoc

```diff
@@ -196,41 +185,26 @@ include::examples/generated/ispn-site-b.yaml[tag=infinispan-crossdc]
 +
 {project_name} requires the following caches to be present: `sessions`, `actionTokens`, `authenticationSessions`, `offlineSessions`, `clientSessions`, `offlineClientSessions`, `loginFailures`, and `work`.
 +
-Due to the use of persistent-user-sessions, the configuration of the caches `sessions`, `offlineSessions`, `clientSessions` and `offlineClientSessions` is different to the `authenticationSessions`, `actionTokens`,
-`loginFailures` and `work` caches. In this section we detail how to configure both types of cache for multi-site deployments.
-+
 The {jdgserver_name} {infinispan-operator-docs}#creating-caches[Cache CR] allows deploying the caches in the {jdgserver_name} cluster.
 Cross-site needs to be enabled per cache as documented by {infinispan-xsite-docs}[Cross Site Documentation].
 The documentation contains more details about the options used by this {section}.
 The following example shows the `Cache` CR for `{site-a}`.
 +
 --
-.`sessions`, `offlineSessions`, `clientSessions` and `offlineClientSessions` caches in `{site-a}`
+. In `{site-a}` create a `Cache` CR for each of the caches mentioned above with the following content.
```

Comment:
Do we still need to create the `*sessions*` caches if they're not used?

Reply:
Let's address this here: https://github.com/keycloak/keycloak/issues/32131